controllers/applicationvolumereplication_controller.go

```diff
@@ -116,19 +115,28 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
 	if err := r.updateAVRStatus(ctx, avr, placementDecisions); err != nil {
 		logger.Error(err, "failed to update status")
 
-		return ctrl.Result{}, err
+		return ctrl.Result{Requeue: true}, err
 	}
 
 	logger.Info("completed manifestwork for subscriptions")
 
-	return ctrl.Result{}, nil
+	return ctrl.Result{Requeue: requeue}, nil
 }
 
 func (r *ApplicationVolumeReplicationReconciler) processSubscriptionList(
-	subscriptionList *subv1.SubscriptionList) ramendrv1alpha1.SubscriptionPlacementDecisionMap {
+	subscriptionList *subv1.SubscriptionList) (
+	ramendrv1alpha1.SubscriptionPlacementDecisionMap, bool) {
 	placementDecisions := ramendrv1alpha1.SubscriptionPlacementDecisionMap{}
 
+	requeue := false
+
 	for _, subscription := range subscriptionList.Items {
+		if subscription.Status.Phase == subv1.SubscriptionSubscribed {
```

Comment:
Actually, this was kind of handled in the selectPlacementDecisions by checking for the subv1.SubscriptionPropagated. But this does not hurt having here as well, so I'll going to incorporate this in the my pause/unpuase PR